packages/core/src/builder.class.ts

```diff
@@ -2633,6 +2633,10 @@ export class Builder {
       }
     }
 
+    if (apiEndpoint === 'content' && queue[0].model === 'symbol') {
+      queryParams['query.id'] = queue[0].entry;
+    }
+
```

Comment:
We should set this in `Symbol.tsx` instead. I think it can go in here: https://github.com/BuilderIO/builder/blob/bff68585eca182176952265a4a0c1f52caabb3cc/packages/react/src/blocks/Symbol.tsx#L159-L165

The reason for that is because the model name might be something other than `symbol`

Reply:
The change is working as expected. Here is a loom with a demo of the changes https://www.loom.com/share/083d6711f94048ce8b88c514c71012e4